packages/swapper/src/swappers/JupiterSwapper/utils/jupiterService.ts

```diff
@@ -0,0 +1,20 @@
+import { SwapperName } from '../../../types'
+import { createCache, makeSwapperAxiosServiceMonadic } from '../../../utils'
+
+const maxAgeMillis = 15 * 1000 // 15 seconds ttl for all except cached to get fresh status updates
+const cachedUrls: string[] = []
```

Comment:
This is copied over from Chainflip but doesn't look correct, no URLs are cached here

```suggestion
const maxAgeMillis = 15 * 1000
const cachedUrls: string[] = []
```

Reply:
Changed in [4e76e81](https://github.com/shapeshift/web/pull/8120/commits/4e76e8115223d91bd602f7d9ceed80c94c77ea15)